packages/template-retail-react-app/app/pages/checkout/confirmation.test.js

```diff
@@ -112,3 +112,138 @@ test('Create Account form - successful submission results in redirect to the Acc
         expect(window.location.pathname).toBe('/uk/en-GB/account')
     })
 })
+
+test('Create Account form - phone number from order shipping address is saved to customer profile', async () => {
```

Comment:
Please clean up comments.  You can group all the tests for account form using describe('Account form', () => {
  test('saves phone number from shipping address to customer', () => {})

})